content/specific-skill-success-criteria/for-loops/_index.md

```diff
@@ -14,7 +14,7 @@ Many people do pattern matching instead of understanding. Here are some common t
 
 ## Note about submission format
 
-On Tilde you'll notice that this card is asking for a link submission. **Please don't worry about submitting a link**. You will be assessed according to {{% contentlink path="specific-skill-success-criteria/introduction-to-assessments" %}}
+On Tilde you'll notice that this card is asking for link submission. **Please don't worry about submitting a link**. You will be assessed according to {{% contentlink path="specific-skill-success-criteria/introduction-to-assessments" %}}
```

Comment:
```suggestion
On Tilde you'll notice that this card is asking for a link submission. **Please don't worry about submitting a link**. You will be assessed according to {{% contentlink path="specific-skill-success-criteria/introduction-to-assessments" %}}
```

Reply:
I removed it cause Grammarly throws an error(indefinite article may be redundant when used with the uncountable noun submission)